### PR TITLE
Set _isDarkMode based upon events

### PR DESCRIPTION
**Pull request by TomasMorton**

### CODE REVIEW

File: Assets/Scripts/Entities/Enemy.cs [hunk 1]
```diff
@@ -137,5 +146,15 @@ private void OnDrawGizmos()
             Gizmos.color = Color.yellow;
             Gizmos.DrawWireSphere(transform.position, aggravationRange);
         }
+
+        private void SetupEventSubscriptions()
+        {
+            _gameController.timer.OnPhaseChange.AddListener(SetWorldPhase);
```
Comment (TomasMorton) [inline Assets/Scripts/Entities/Enemy.cs:153]: Do we need to remove the listener in the Die() function?

Comment (baconeta) [reply Assets/Scripts/Entities/Enemy.cs:153]: The answer to your question, is yes. We do need to remove it. Will you add it?

Comment (TomasMorton) [reply Assets/Scripts/Entities/Enemy.cs:153]: Sweet, done. Not tested though as I can't attack?

Comment (baconeta) [reply Assets/Scripts/Entities/Enemy.cs:153]: Yeah attacking is in progress XD

**Pull request merged**